DFC.App.Help.IntegrationTests/ServiceTests/HelpPageServiceTests/HelpPageServiceGetByNameTests.cs

```diff
@@ -30,22 +28,32 @@ public async Task HelpPageServiceGetByNameReturnsSuccessWhenHelpPageExists()
             var result = await helpPageService.GetByNameAsync(helpPageModel.CanonicalName).ConfigureAwait(false);
 
             // assert
-            result.DocumentId.Should().Be(helpPageModel.DocumentId);
-            result.CanonicalName.Should().Be(helpPageModel.CanonicalName);
+            Assert.True(result.DocumentId == helpPageModel.DocumentId
+                        && result.CanonicalName.Equals(helpPageModel.CanonicalName, StringComparison.InvariantCulture));
         }
 
         [Test]
         [Category("HelpPageService.GetByName")]
         public async Task HelpPageServiceGetByNameReturnsNullWhenHelpPageDoesNotExist()
+        {
```

Comment:
Add AAA comments to identify the three distinct parts of the test (arrange, act assert) to make it obvious at a glance what is being actioned and what is being asserted.

Reply:
Updated as suggested. Though I've usually used spacing/blank line to separate these parts to avoid polluting with comments